teamscale-client/src/main/java/com/teamscale/client/TeamscaleClient.java

```diff
@@ -64,14 +65,15 @@ public TeamscaleClient(String baseUrl, String user, String accessToken, String p
 	private Response<List<PrioritizableTestCluster>> getImpactedTestsOnce(List<ClusteredTestDetails> testList,
 																		  Long baseline,
 																		  CommitDescriptor endCommit,
-																		  String partition) throws IOException {
+																		  String partition,
+																		  boolean includeNonImpacted) throws IOException {
```

Comment:
What is happening with these empty lines? Is the formatter disabled?

Reply:
You need to scroll to the right 😉
We use the default formatter of intellij here and it aligns the parameters with the opening parenthesis.